src/assets/js/archive.js

```diff
@@ -205,7 +205,7 @@ function updateReleaseDownloadButton(releases, base_url, os, arch = '') {
     element.textContent = archiveFilename;
   });
 
-  const genericDownloadLink = document.querySelectorAll('.download-latest-link-filename');
+  const genericDownloadLink = document.querySelectorAll(`.download-latest-link-filename`);
```

Comment:
I'm curious why this change in string markers. From memory, backticks in JS are meant for string interpolation, but I don't see any interpolation happening.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals